pkg/apis/applicationnetworking/v1alpha1/accesslogpolicy_types.go

```diff
@@ -45,9 +46,8 @@ type AccessLogPolicySpec struct {
 	// Supported values are S3 Bucket, CloudWatch Log Group, and Firehose Delivery Stream ARNs.
 	//
 	// Changes to this value results in replacement of the VPC Lattice Access Log Subscription.
-	// +optional
 	// +kubebuilder:validation:Pattern=`^arn(:[a-z0-9]+([.-][a-z0-9]+)*){2}(:([a-z0-9]+([.-][a-z0-9]+)*)?){2}:([^/].*)?`
-	DestinationArn *string `json:"destinationArn,omitempty"`
+	DestinationArn *string `json:"destinationArn"`
```

Comment:
Since we use pointer everywhere else, I opted to stay consistent with the other places. We also discussed offline and determined that there's no harm in keeping the pointer, as long as we remove omitempty.